diff --git a/cmake/Dotnet.cmake b/cmake/Dotnet.cmake
index 1ab7ecd..7470015 100644
--- a/cmake/Dotnet.cmake
+++ b/cmake/Dotnet.cmake
@@ -49,6 +49,7 @@ function(LinkDotnetNetHost DOTNET TARGET)
 		elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i386" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i686")
 			set(DOTNET_ARCH x86)
-		elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "ARM64" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64")
+		elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "ARM64" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64" OR
+			${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm64")
 			set(DOTNET_ARCH arm64)
 		else()
 			message(FATAL_ERROR "Unable to determine appropriate nethost architecture")
